src/multiselect_contextmenu.js

```diff
@@ -837,13 +930,279 @@ const updateToMultiCopyToBackpack =
+        const comment = workspace.getCommentById(id);
+        if (comment) {
+          if (copyOptions.check(comment)) {
+            workableCommentsLength++;
+          }
+        }
+      });
+      if (workableCommentsLength <= 1) {
+        return Blockly.Msg['CROSS_TAB_COPY']?
+            Blockly.Msg['CROSS_TAB_COPY'] : 'Copy';
+      } else {
+        return Blockly.Msg['CROSS_TAB_COPY_X_COMMENTS']?
+            Blockly.Msg['CROSS_TAB_COMMENTS'].replace(
+                '%1', workableCommentsLength) :
+            (Blockly.Msg['CROSS_TAB_COPY']?
+                    Blockly.Msg['CROSS_TAB_COPY'] : 'Copy'
+            ) + ' (' +
+            workableCommentsLength + ')';
+      }
+    },
+    preconditionFn: function(scope) {
+      const workspace = scope.comment.workspace;
+      if (workspace.options.readOnly && !useCopyPasteCrossTab) {
+        return 'hidden';
+      }
+
+      const dragSelection = dragSelectionWeakMap.get(workspace);
+
+      // Fix the context menu error for backpack plugin
+      if (dragSelection === undefined) {
+        return 'hidden';
+      }
+
+      if (!dragSelection.size) {
+        if (copyOptions.check(scope.comment)) {
+          return 'enabled';
+        } else {
+          return 'disabled';
+        }
+      }
+      for (const id of dragSelection) {
+        const comment = workspace.getCommentById(id);
+        if (comment) {
+          if (copyOptions.check(comment)) {
+            return 'enabled';
+          }
+        }
+      }
+      return 'disabled';
+    },
+    check: function(comment) {
+      return comment && comment.isDeletable() &&
+          comment.isMovable();
+    },
+    callback: function(scope) {
+      const workspace = scope.comment.workspace;
+      copyData.clear();
+      workspace.hideChaff();
+
+      const apply = function(comm) {
+        if (copyOptions.check(comm)) {
+          copyData.add(JSON.stringify(comm.toCopyData()));
+        }
+      };
+
+      const selected = Blockly.common.getSelected();
+      const dragSelection = dragSelectionWeakMap.get(workspace);
+      Blockly.Events.setGroup(true);
+
+      // Handle the case where MultiselectDraggable is in use
+      if (selected && selected instanceof MultiselectDraggable) {
+        for (const element of selected.subDraggables) {
+          if (element[0] instanceof Blockly.comments.RenderedWorkspaceComment) {
+            apply(element[0]);
+          }
+        }
+      } else if (!dragSelection.size) {
+        apply(scope.comment);
+      }
+
+      if (useCopyPasteCrossTab) {
+        dataCopyToStorage();
+      }
+      Blockly.Events.setGroup(false);
+      return true;
+    },
+    scopeType: Blockly.ContextMenuRegistry.ScopeType.COMMENT,
+    id: 'commentCopyToStorage',
+    weight: 0,
+  };
+  if (Blockly.ContextMenuRegistry.registry.getItem('commentCopyToStorage') !==
+      null) {
+    Blockly.ContextMenuRegistry.registry.unregister('commentCopyToStorage');
+  }
```

Comment:
This can be removed, we don't have `commentCopyToStorage` in cross-tab-copy-paste plugin.